### PR TITLE
Local deployment docs for local deployment with ADB

### PR DESCRIPTION
**Pull request by zeroZshadow**
#### Description
Rewrote Android local deployment documentation to use ADB.

#### Tests
Ran local improbadoc and checked that everything looks OK.

### CODE REVIEW

File: docs/content/mobile/android/local-deploy.md [hunk 1]
```diff
@@ -1,73 +1,69 @@
...
+
+To allow a mobile phone to connect to your local deployment, you will need to make the following configuration steps.
+
+1. Have your project opened in the Unity Editor.
+1. Open the configuration window through **SpatialOS** > **GDK Tools configuration**.
+1. Fill your local machine's IP into the **Runtime IP for local deployment** field.
+1. Click **Save**, and close the window.
+
+From this point on, all local deployments launched through the editor will allow your mobile phone to connect to it.
+
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]:
```
# Prepare your project to connect to a local deployment

To connect your Android device to a local deployment, follow these steps:

 1. Open your project in the Unity Editor.
1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
1. Fill your [local machine's IP](This needs a link on how to find it) into the **Runtime IP for local deployment** field.
1. Click **Save**, and close the window.

This ensures that any local deployment that is launched via the Unity Editor is set up correctly to connect to your Android device. 
```

always try to use active voice, Unity Editor is always capitalized.

File: docs/content/mobile/android/local-deploy.md [hunk 2]
```diff
@@ -1,73 +1,69 @@
...
-  1. Click **Build and Run**.
-
-    > For subsequent runs using the same Emulator, you can just select **File** > **Build and Run**.
-
-  1. Once the game is deployed and started on the Emulator, you see an empty text field and a **Connect** button: Select **Connect**.
-
-    > You don’t need to enter anything in the text field.
-
-  1. Play the game on the Emulator.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your android worker, and wait for the build to complete.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: Android needs to be capitalized, I believe

File: docs/content/mobile/android/local-deploy.md [hunk 3]
```diff
@@ -1,73 +1,69 @@
...
 
 ## Connecting your Android emulator to a local deployment
 
-  1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
+1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
 
     > Ensure to choose the same CPU architecture for your virtual machine as your development computer. If you don’t, you will get warning messages as mis-matched CPU architecture affects performance.
+1. [Setup your local deployment.](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. Start the local deployment from the Unity Editor though **SpatialOS** > **Local launch**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: "through" instead of "though"

"your local deployment" instead of "the local deployment" -> just makes it a bit clearer that this is happening on their side

File: docs/content/mobile/android/local-deploy.md [hunk 4]
```diff
@@ -1,73 +1,69 @@
...
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Setup your local deployment](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. Start the local deployment from the Unity Editor though **SpatialOS** > **Local launch**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: through instead of though

File: docs/content/mobile/android/local-deploy.md [hunk 5]
```diff
@@ -1,73 +1,69 @@
...
     > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
 
-  1. In the Unity Editor, go to **File** > **Build Settings**. Choose the mobile device that you connected to your computer from the drop-down menu in the **Run Device** field.
-  1. Click **Build and Run**.
-
-    > For subsequent runs you can just select **File** > **Build and Run**.
-
-  1. Once the game is running on your device, you see an empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
-  1. Play the game on your mobile device.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your android worker, and wait for the build to complete.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: "through" instead of "though"

"your local deployment" instead of "the local deployment" -> just makes it a bit clearer that this is happening on their side

File: docs/content/mobile/android/local-deploy.md [hunk 6]
```diff
@@ -1,73 +1,69 @@
...
+1. Click **SpatialOS** > **Launch mobile client** > **Android Device**.
+1. Play the game on the device.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: on your mobile device

File: docs/content/mobile/android/local-deploy.md (hunk 1 above)
Comment (ElleEss) [reply docs/content/mobile/android/local-deploy.md]: <3

Comment (ElleEss) [reply docs/content/mobile/android/local-deploy.md]: +1 on Jess's suggestions. A couple of additions...

1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
...
These steps ensure that any local deployment you launch via the Unity Editor is set up to connect to your Android device correctly.

File: docs/content/mobile/android/local-deploy.md [hunk 7]
```diff
@@ -1,73 +1,69 @@
...
 
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+1. You should now see your Unity Editor game view mirrored on your Android device.
 
 ## Connecting your Android emulator to a local deployment
 
-  1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
+1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
 
     > Ensure to choose the same CPU architecture for your virtual machine as your development computer. If you don’t, you will get warning messages as mis-matched CPU architecture affects performance.
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]: Ensure to choose the same
```suggestion
    > Ensure you choose the same CPU architecture for your virtual machine as your development computer. If you don’t, you will get warning messages as mismatched CPU architecture affects performance.
```

File: docs/content/mobile/android/local-deploy.md (hunk 3 above)
Comment (ElleEss) [reply docs/content/mobile/android/local-deploy.md]: <3 <3 <3

Comment (ElleEss) [reply docs/content/mobile/android/local-deploy.md]: Have added this and another suggestion below. I would probably use "your Unity Editor" when writing but I note that most of this doc says "the Unity Editor". Either is fine but "your Unity Editor" sounds more comfortable as it's your version on your machine.

Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. Start your local deployment from the Unity Editor's menu: **SpatialOS** > **Local launch**.
```

File: docs/content/mobile/android/local-deploy.md (hunk 2 above)
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your Android worker, and wait for the build to complete.
```

Comment (ElleEss) [reply docs/content/mobile/android/local-deploy.md]: There we are.

File: docs/content/mobile/android/local-deploy.md [hunk 8]
```diff
@@ -1,73 +1,69 @@
...
-  1. [Build your server-workers.]({{urlRoot}}/content/build)
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Setup your local deployment](#setup-local-deployment-for-mobile-phone-connections)
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. [Set up your local deployment](#setup-local-deployment-for-mobile-phone-connections)
```

File: docs/content/mobile/android/local-deploy.md [hunk 9]
```diff
@@ -1,73 +1,69 @@
...
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+1. You should now see your Unity Editor game view mirrored on your Android device.
 
 ## Connecting your Android emulator to a local deployment
 
-  1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
+1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
 
     > Ensure to choose the same CPU architecture for your virtual machine as your development computer. If you don’t, you will get warning messages as mis-matched CPU architecture affects performance.
+1. [Setup your local deployment.](#setup-local-deployment-for-mobile-phone-connections)
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. [Set up your local deployment.](#setup-local-deployment-for-mobile-phone-connections)
```
As set up is a verb here, it's two words. (Yes, I know: ENGLISH!!!)

File: docs/content/mobile/android/local-deploy.md [hunk 10]
```diff
@@ -1,73 +1,69 @@
...
-
-    > For subsequent runs using the same Emulator, you can just select **File** > **Build and Run**.
-
-  1. Once the game is deployed and started on the Emulator, you see an empty text field and a **Connect** button: Select **Connect**.
-
-    > You don’t need to enter anything in the text field.
-
-  1. Play the game on the Emulator.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your android worker, and wait for the build to complete.
+1. Click **SpatialOS** > **Launch mobile client** > **Android Device**.
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. Still in the Unity Editor, from the menu, select **SpatialOS** > **Launch mobile client** > **Android Device**.
```

File: docs/content/mobile/android/local-deploy.md (hunk 5 above)
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your Android worker, and wait for the build to complete.
```

File: docs/content/mobile/android/local-deploy.md (hunk 6 above)
Comment (ElleEss) [reply docs/content/mobile/android/local-deploy.md]: Thumbs up.

File: docs/content/mobile/android/local-deploy.md [hunk 11]
```diff
@@ -1,73 +1,69 @@
 [//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
 
+# Prepare your project to connect to a local deployment
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: this section should be below the "Before reading this document..." and there should only be one header 1 as that describes the overall intent of the doc.

So try to structure it like this:

```
 # Connecting to a local deployment

Before reading...

## Prepare your project to connect

...

## Connect to a local deployment

After everything has been set up, you have now three options on how to connect.

## Connecting your Android device to a local deployment using Unity Remote

..

## Connecting your Android emulator to a local deployment

..

## Connecting your Android device to a local deployment

..

```

Need to check how that would look, it might even be better to have expandable tabs for the three options.

Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: and get rid of the todo at the top as we don't mention mobile_launch.json anymore

File: docs/content/mobile/android/local-deploy.md [hunk 12]
```diff
@@ -1,73 +1,67 @@
...
-  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
-  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+1. In the **Unity Remote** section, select the drop-down menu beside the **Device** option and select **Any Android Device**.
+1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: just noticed that the "Select" should not be capitalized

Comment (jessicafalk) [reply docs/content/mobile/android/local-deploy.md]: and let's rephrase it to 

```suggestion
1. In the Unity Editor, select **SpatialOS** > **Local launch** to start your local deployment.
```

File: docs/content/mobile/android/local-deploy.md [hunk 13]
```diff
@@ -1,73 +1,67 @@
...
+1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
 
-  1. [Build your server-workers.]({{urlRoot}}/content/build)
-  1. You need to know the local IP address of your computer to connect. [This page](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) (on the Lifehacker website)  describes how you can find your external and local IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`.  (Where `<your-local-ip>` is the IP address you just located.)
+    > Ensure you choose the same CPU architecture for your virtual machine as your development computer. If you don’t, you will get warning messages as mismatched CPU architecture affects performance.
+1. [Set up your local deployment.](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. Start your local deployment from the Unity Editor's menu: **SpatialOS** > **Local launch**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, select **SpatialOS** > **Local launch** to start your local deployment.
```

File: docs/content/mobile/android/local-deploy.md [hunk 14]
```diff
@@ -1,73 +1,67 @@
...
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Set up your local deployment](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. Start your local deployment from the Unity Editor's menu: **SpatialOS** > **Local launch**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, select **SpatialOS** > **Local launch** to start your local deployment.
```

File: docs/content/mobile/android/local-deploy.md [hunk 15]
```diff
@@ -1,73 +1,67 @@
...
 
-  1. In the Unity Editor, go to **File** > **Build Settings**. Choose the mobile device that you connected to your computer from the drop-down menu in the **Run Device** field.
-  1. Click **Build and Run**.
-
-    > For subsequent runs you can just select **File** > **Build and Run**.
-
-  1. Once the game is running on your device, you see an empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
-  1. Play the game on your mobile device.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your Android worker, and wait for the build to complete.
+1. Now select **SpatialOS** > **Launch mobile client** > **Android Device**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, select **SpatialOS** > **Launch mobile client** > **Android Device** to start your Android client.
```

File: docs/content/mobile/android/local-deploy.md [hunk 16]
```diff
@@ -1,73 +1,67 @@
...
-
-    > For subsequent runs using the same Emulator, you can just select **File** > **Build and Run**.
-
-  1. Once the game is deployed and started on the Emulator, you see an empty text field and a **Connect** button: Select **Connect**.
-
-    > You don’t need to enter anything in the text field.
-
-  1. Play the game on the Emulator.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your Android worker, and wait for the build to complete.
+1. Now select **SpatialOS** > **Launch mobile client** > **Android Device**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, select **SpatialOS** > **Launch mobile client** > **Android Device** to start your Android client.
```

File: docs/content/mobile/android/local-deploy.md [hunk 17]
```diff
@@ -1,73 +1,67 @@
...
-  1. [Build your server-workers.]({{urlRoot}}/content/build)
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Set up your local deployment](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: would swap these two to have the two local deployment steps together

File: docs/content/mobile/android/local-deploy.md [hunk 18]
```diff
@@ -1,73 +1,67 @@
...
+1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+1. In the **Unity Remote** section, select the drop-down menu beside the **Device** option and select **Any Android Device**.
+1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
 
     > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
 
-  1. In the Editor’s Game view, select **Play**.
+1. In the Editor’s Game view, select **Play**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor’s Game view, select **Play**.
```

File: docs/content/mobile/android/local-deploy.md [hunk 19]
```diff
@@ -1,73 +1,67 @@
...
 
     > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
 
-  1. In the Editor’s Game view, select **Play**.
+1. In the Unity Editor’s Game view, select **Play**.
 
     > You can change the resolution of the Game view in your Unity Editor to make sure it does not appear stretched on your mobile device. Choose the resolution that’s identical to your mobile device to produce the best results.
 
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+1. You should now see your Unity Editor game view mirrored on your Android device.
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. You should now see your Unity Editor's Game view mirrored on your Android device.
```

File: docs/content/mobile/android/local-deploy.md [hunk 20]
```diff
@@ -1,73 +1,67 @@
-[//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
-
 # Connecting to a local deployment
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
# Connect to a local deployment
```

We decided to lose the gerunds (the "ing") for the titles in GDK docs. Just noticed this!

File: docs/content/mobile/android/local-deploy.md [hunk 21]
```diff
@@ -1,73 +1,67 @@
...
+To connect your Android device to a local deployment, follow these steps:
+
+1. Open your project in the Unity Editor.
+1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
+1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
+1. Select **Save**, and close the window.
+
+This ensures that any local deployment that is launched via the Unity Editor is set up correctly to connect to your Android device.
 
 ## Connecting your Android device to a local deployment using Unity Remote
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
## Connect your Android device to a local deployment using Unity Remote
```
To make it not a gerund. :-)

File: docs/content/mobile/android/local-deploy.md [hunk 22]
```diff
@@ -1,73 +1,67 @@
...
 
-  1. In the Editor’s Game view, select **Play**.
+1. In the Unity Editor’s Game view, select **Play**.
 
     > You can change the resolution of the Game view in your Unity Editor to make sure it does not appear stretched on your mobile device. Choose the resolution that’s identical to your mobile device to produce the best results.
 
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+1. You should now see your Unity Editor game view mirrored on your Android device.
 
 ## Connecting your Android emulator to a local deployment
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
## Connect your Android Emulator to a local deployment
```

File: docs/content/mobile/android/local-deploy.md [hunk 23]
```diff
@@ -1,73 +1,67 @@
...
-  1. Once the game is deployed and started on the Emulator, you see an empty text field and a **Connect** button: Select **Connect**.
-
-    > You don’t need to enter anything in the text field.
-
-  1. Play the game on the Emulator.
+1. Still in the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your Android worker, and wait for the build to complete.
+1. Select **SpatialOS** > **Launch mobile client** > **Android Device**.
+1. Play the game on the Emulator.
 
 ## Connecting your Android device to a local deployment
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
## Connect your Android device to a local deployment
```

File: docs/content/mobile/android/local-deploy.md [hunk 24]
```diff
@@ -1,73 +1,67 @@
...
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. [Set up your local deployment](#prepare-your-project-to-connect-to-a-local-deployment)
+1. In the Unity Editor, select **SpatialOS** > **Local launch** to start your local deployment
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. In the Unity Editor, select **SpatialOS** > **Local launch** to start your local deployment.
```

File: docs/content/mobile/android/local-deploy.md [hunk 25]
```diff
@@ -1,73 +1,67 @@
...
-  1. [Build your server-workers.]({{urlRoot}}/content/build)
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. [Set up your local deployment](#prepare-your-project-to-connect-to-a-local-deployment)
```
Comment (ElleEss) [inline docs/content/mobile/android/local-deploy.md]:
```suggestion
1. [Set up your local deployment](#prepare-your-project-to-connect-to-a-local-deployment).
```

**Comment by ElleEss** [comment]
Also, made a suggestion to make it clear that these are three different ways to do the same thing. (If I have understood it correctly?)

File: docs/content/mobile/android/local-deploy.md [hunk 26]
```diff
@@ -1,73 +1,67 @@
...
+## Prepare your project to connect to a local deployment
+
+To connect your Android device to a local deployment, follow these steps:
+
+1. Open your project in the Unity Editor.
+1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
+1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
+1. Select **Save**, and close the window.
+
+This ensures that any local deployment that is launched via the Unity Editor is set up correctly to connect to your Android device.
```
[the remark this answers is not in the document]
Comment (zeroZshadow) [reply docs/content/mobile/android/local-deploy.md]: I'm not sure what you suggestion is here. Do you want me to simply move this bit to be above "To connect your ...."?

File: docs/content/mobile/android/local-deploy.md [hunk 27]
```diff
@@ -1,73 +1,65 @@
...
-  1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
-  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
-  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
-  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
-  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+## Prepare your project to connect to a local deployment
 
-    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+To connect your Android device to a local deployment, you need to prepare; follow the steps below to ensure any local deployment you launch via the Unity Editor is correctly set up.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: `To connect your Android device to a local deployment, you need to prepare;` -> what do you need to prepare?

File: docs/content/mobile/android/local-deploy.md [hunk 28]
```diff
@@ -1,73 +1,65 @@
...
 
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+You need the Unity Remote app for this. See the [Unity documentation](https://docs.unity3d.com/Manual/UnityRemote5.html) for details.
 
-## Connecting your Android emulator to a local deployment
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. In the Unity Editor, open the project that you want to deploy and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+1. Open the Scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+1. In **Editor Settings** window's **Unity Remote** section, set the **Device** setting to **Any Android Device**.
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: Just noticed that you completely rewrote this sentence.
- a `the` is missing in `In **Editor Settings** ..`
- where is this Editor settings window?
